server/app/services/question/PrimaryApplicantInfoTag.java

```diff
@@ -0,0 +1,105 @@
+package services.question;
+
+import com.google.common.collect.ImmutableList;
+import models.QuestionTag;
+import services.question.types.QuestionType;
+
+/**
+ * Defines tags that can be applied to a question to denote the answer is a primary piece of
+ * information about the applicant. These are 1:1 with particular QuestionTags.
```

Comment:
suggestion (non-blocking):
```suggestion
 * information about the applicant. These are 1:1 with {@link QuestionTag}s that start with `PRIMARY_APPLICANT`.
```